frontend/app/src/components/status/update/AssetUpdateMessage.vue

```diff
@@ -16,6 +16,7 @@ const emit = defineEmits<{
 const { versions } = toRefs(props);
 
 const partial: Ref<boolean> = ref(false);
+const maxToVersion: Ref<number> = ref(0);
```

Comment:
I found bug before, if you can upgrade from v 14 to 16.
And you activate the partial, set lowest version possible (14), and then disabled the partial
If I press update, it should upgrade to the default version (to the max / 16), but instead the value used is 14 (the partial value, which shouldn't be used because it's been disabled)

So what I do is, if we disable the partial, I set it back to the default version

Reply:
Can't we just update the upToVersion to the remote version?

Reply:
oh yeah, just realize I can do that too, will update it

Reply:
updated